nuxt.config.js

```diff
@@ -1,4 +1,5 @@
 const env = require('dotenv').config().parsed
+const ssrDirectives = require('./directives/ssr/index.js')
```

Comment:
there is no need to require index, simply rewuire directives/ssr and it will work